pkg/security/controller/controller.go

```diff
@@ -65,6 +66,7 @@ func (c *Allocation) Next(ns *kapi.Namespace) error {
 	}
 	tx.Add(func() error { return c.uid.Release(block) })
 	ns.Annotations[security.UIDRangeAnnotation] = block.String()
+	ns.Annotations[security.FSGroupAnnotation] = strconv.FormatUint(uint64(block.Start), 10)
```

Comment:
@smarterclayton Thoughts on this?  If FSGroup only has to be unique to a namespace, I think this will work.  If we need to isolate pods in the same NS from one another we will need to do a real allocation